### ISSUE TITLE
[cspacer] channels

### ISSUE THREAD

**Issue by preterive** [mumble-voip/mumble#3384]
Hello,
i wish spacer channels like on Teamspeak for example: [cspacer]

![image](https://user-images.githubusercontent.com/31483348/37969280-a0d3924e-31d0-11e8-9688-efada1bd1a13.png)

**Comment by maxhambraeus** [comment]
I think this feature, and a few others mentioned in #3608, would do a lot for the general usability and user-friendlyness of mumble. To me this seems pretty straight-forward to implement, but I don't know anything about the codebase.

**Comment by Hartmnt** [comment]
> Effectively, this should already be achievable without code changes but having UI support for setting everything up as required with one click seems desirable.

True, but I would argue that this feature would be a lot *nicer* to use, if there were code changes. Let's say there are some hard-coded channel names for space channels e.g. ``[cspacer]``
In the client, we could use an item delegate thingy to draw a nice line or whatever without necessarily depending on fonts or Unicode. Instead of not having the permission to enter, the client would not even give the user the option to join the channel. With regards to accessibility, the clients using screen readers could read "spacer channel" instead of "dashdashdashdashdash" ;)
There are probably some server side things to consider, too.

**Comment by danielwerg** [comment]
I thought there was workaround for this but using "―" or other unicode symbols I tried gives "Denied: Invalid channel name." error.

**Comment by Krzmbrzl** [comment]
The characters allowed in channel names are determined in the server's configuration.

**Comment by danielwerg** [comment]
Oh, you are right, default:

```ini
channelname=[ \\-=\\w\\#\\[\\]\\{\\}\\(\\)\\@\\|]+
```

**Comment by Krzmbrzl** [comment]
> Is there any downsides to setting it to .*?

Yes, you'll open the door for Unicode homoglyph spoofing. Though for channel names, it's probably a lot less of an issue than for user names. :shrug:
